### ISSUE TITLE
ADD source type for lat/long links for loughborough

### ISSUE THREAD

**Issue by polly64** [colouring-cities/colouring-britain#275]
@matkoniecz just need to add sourec type and source link /methods for how we got these

**Comment by matkoniecz** [comment]
What you mean by "lat/long links for loughborough"?

**Comment by polly64** [comment]
@matkoniecz sorry just means can we give source for any location data

**Comment by matkoniecz** [comment]
Sorry, I still do not understand. Can you attach a screenshot with marked location where it would appear? Is it unfilled place with space for info or about adding new credit location?

**Comment by polly64** [comment]
@mateusz attached, also there is a nice new feature Dresden has created to allow you to click top right of map and show where you are 
<img width="458" alt="Screenshot 2024-03-28 at 15 44 38" src="https://github.com/colouring-cities/colouring-britain/assets/42236514/da389b91-4ec8-4873-8f95-938f751399be">

**Comment by polly64** [comment]
yes about adding bulk data if we are using same source

**Comment by matkoniecz** [comment]
Thanks!

So source type for INSPIRE/Local map merger should be: "Current government record/database".

And links: https://use-land-property-data.service.gov.uk/datasets/inspire/download https://osdatahub.os.uk/downloads/open/OpenMapLocal

But for master map polygons (London/Lboro) we cannot really set "Open property coordinates" as to assign them to building we would match them based on building polygon geometry from master map.

**Comment by polly64** [comment]
@matkoniecz  yes I see . so first credit is right. For second I think we should change the name 'open footprints, as actually none of our footprints will be able to accessed in the corp because we want them behind a fire wall in  our system for security. So let's call them something else like. "Base building footprints/polygons used for data collation, verification and visualisation" what do you think?

**Comment by matkoniecz** [comment]
That makes sense! Should the change be applied to main site or just colouring-britain?

Can we share centers of Master Map polygons or other coordinates based on match with them?

**Comment by polly64** [comment]
main site please
